src/Microsoft.SqlTools.ServiceLayer/EditData/Contracts/EditRevertRowRequest.cs

```diff
@@ -21,6 +21,10 @@ public class EditRevertRowParams : RowOperationParams
     /// </summary>
     public class EditRevertRowResult
     {
+        /// <summary>
+        /// The row after the revert was applied
```

Comment:
Is this the same as the original, unedited row?  If so, that might be good clarification.